File: Sistema/src/app/models/SessaoConvite.Model.ts
export class SessaoConviteModel {
  SesCodi: number;
  SesNume: number;
  SesNome?: string;
  SesDesc?: string;
  SesDtHr: Date;
  SesLibe: boolean;
  SesStat: boolean;
  TiSNome?: string;
  GraNome?: string;
  LojCodi: number;
  LojNome?: string;
  LojStat: boolean;
  LojNume?: string;
  LojLogr?: string;
  LojNumL?: string;
  LojBair?: string;
  PotNome?: string;
  PotSigl?: string;
  PotRegu: boolean;
  RitNome?: string;
  CidNome?: string;
  EstSigl?: string;
  LojLogo?: string;
  PotLogo?: string;

  constructor(
    SesCodi: number,
    SesNume: number,
    SesDtHr: Date,
    SesLibe: boolean,
    SesStat: boolean,
    LojCodi: number,
    LojStat: boolean,
    PotRegu: boolean,
    SesNome?: string,
    SesDesc?: string,
    TiSNome?: string,
    GraNome?: string,
    LojNome?: string,
    LojNume?: string,
    LojLogr?: string,
    LojNumL?: string,
    LojBair?: string,
    PotNome?: string,
    PotSigl?: string,
    RitNome?: string,
    CidNome?: string,
    EstSigl?: string,
    LojLogo?: string,
    PotLogo?: string
  ) {
    this.SesCodi = SesCodi;
    this.SesNume = SesNume;
    this.SesNome = SesNome;
    this.SesDesc = SesDesc;
    this.SesDtHr = SesDtHr;
    this.SesLibe = SesLibe;
    this.SesStat = SesStat;
    this.TiSNome = TiSNome;
    this.GraNome = GraNome;
    this.LojCodi = LojCodi;
    this.LojNome = LojNome;
    this.LojStat = LojStat;
    this.LojNume = LojNume;
    this.LojLogr = LojLogr;
    this.LojNumL = LojNumL;
    this.LojBair = LojBair;
    this.PotNome = PotNome;
    this.PotSigl = PotSigl;
    this.PotRegu = PotRegu;
    this.RitNome = RitNome;
    this.CidNome = CidNome;
    this.EstSigl = EstSigl;
    this.LojLogo = LojLogo;
    this.PotLogo = PotLogo;
  }
}

File: API/Models/SessaoConviteModel.cs
namespace API_Visitatus.Models
{
    public class SessaoConviteModel
    {
        public long SesCodi { get; set; }
        public long SesNume { get; set; }
        public string? SesNome { get; set; }
        public string? SesDesc { get; set; }
        public DateTime SesDtHr { get; set; }
        public bool SesLibe { get; set; }
        public bool SesStat { get; set; }
        public string? TiSNome { get; set; }
        public string? GraNome { get; set; }
        public long LojCodi { get; set; }
        public string? LojNome { get; set; }
        public bool LojStat { get; set; }
        public string? LojNume { get; set; }
        public string? LojLogr { get; set; }
        public string? LojNumL { get; set; }
        public string? LojBair { get; set; }
        public string? PotNome { get; set; }
        public string? PotSigl { get; set; }
        public bool PotRegu { get; set; }
        public string? RitNome { get; set; }
        public string? CidNome { get; set; }
        public string? EstSigl { get; set; }
        public List<GestaoAdmAtivaModel>? lstGestaoAdmAtiva{ get; set; }
        public string? LojLogo { get; set; }
        public string? PotLogo { get; set; }
        public bool SesAgap { get; set; }
        public decimal? SesVlAg { get; set; }
    }
}
